### PR TITLE
feat(ng-option): support for disabled attribute

### LINKED ISSUE #236: ng-option: support for `disabled` attribute
`ng-option` currently not supports `disabled` attribute. All attributes (except `selected`, `label`) supported by native html `option` should be supported.

### PR DESCRIPTION
**Pull request by varnastadeus**
closes #236

### CODE REVIEW

File: src/ng-select/ng-select.component.ts [hunk 1]
```diff
@@ -179,12 +182,13 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, AfterVie
     }
 
     ngOnDestroy() {
-        this.changeDetectorRef.detach();
```
Comment (anjmao) [inline src/ng-select/ng-select.component.ts]: Check if it works when using *ngIf on ng-select (example is in reactive forms)

Comment (varnastadeus) [reply src/ng-select/ng-select.component.ts]: @anjmao it works
